data/time_series.go

```diff
@@ -214,6 +215,7 @@ func LongToWide(longFrame *Frame, fillMissing *FillMissing) (*Frame, error) {
 	}
 
 	wideFrame := NewFrame(longFrame.Name, NewField(longFrame.Fields[tsSchema.TimeIndex].Name, nil, []time.Time{}))
+	wideFrame.Meta = longFrame.Meta
```

Comment:
I had punted on this because it doesn't really copy the FrameMeta, rather just a reference to it:

```
type Frame struct {
	// Name is used in some Grafana visualizations.
	Name string

	// Fields are the columns of a frame.
	// All Fields must be of the same the length when marshalling the Frame for transmission.
	Fields []*Field

	// RefID is a property that can be set to match a Frame to its orginating query
	RefID string

	// Meta is metadata about the Frame, and includes space for custom metadata.
	Meta *FrameMeta
}
```

So ideally we have `FrameMeta.Copy()`.  But, we don't need this right now, I think a comment pointing out that a reference to meta is copied is sufficient.

Reply:
Think the comment should be on the function inself (parted of exported docs).